WWDC/SessionSummaryViewController.swift

```diff
@@ -200,8 +200,9 @@ class SessionSummaryViewController: NSViewController {
 
         viewModel.rxSummary.subscribe(onNext: { [weak self] summary in
             guard let self = self else { return }
-            let range = NSRange(location: 0, length: self.summaryTextView.string.count)
```

Comment:
I found that the string count isn't always the length of the text storage, resulting in cases of some mysterious characters at the end of the new summary!